src/main/java/com/github/dockerjava/api/DockerClient.java

```diff
@@ -251,6 +256,48 @@
 
     DisconnectFromNetworkCmd disconnectFromNetworkCmd();
 
+    /**
+     * Enables swarm mode for the docker engine and creates a new swarm cluster
+     *
+     * @since 1.24
+     * @param swarmSpec the specification for the swarm
+     * @return the command
+     */
+    InitializeSwarmCmd initializeSwarmCmd(SwarmSpec swarmSpec);
```

Comment:
Dind will have performance issues + port exposure issues. I would at all suggest to migrate to Junit with `@Rule` and `@Resources` as i already have a lot of working solutions in yet-another-docker-plugin for it.